<ui version="4.0" >
 <class>UIProjectsViewer</class>
 <widget class="QWidget" name="UIProjectsViewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QTreeView" name="tvProject" />
       </item>
       <item>
        <widget class="QLabel" name="lProject" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbFilter" >
         <property name="text" >
          <string>Filter On/Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="teOutput" >
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbWrite" >
       <property name="text" >
        <string>Write Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSettings" >
       <property name="text" >
        <string>Project Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
